base/shared/src/main/scala/scalaz/typeclass/ApplicativeSyntax.scala

```diff
@@ -8,10 +8,15 @@ import language.experimental.macros
 
 trait ApplicativeSyntax {
   implicit def applicativeOpsA[A](a: A): ApplicativeSyntax.OpsA[A] = new ApplicativeSyntax.OpsA(a)
+  implicit def applicativeOps[A, F[_]: Applicative](fa: F[A]): ApplicativeSyntax.Ops[A, F] = new ApplicativeSyntax.Ops(fa)
 }
 
 object ApplicativeSyntax {
   class OpsA[A](@silent a: A) {
     def pure[F[_]: Applicative]: F[A] = macro meta.IdOps.id_1
   }
+
+  class Ops[A, F[_]](@silent fa: F[A])(implicit F: Applicative[F]) {
+    def map[B](f: A => B): F[B] = F.apply.functor.map(fa)(f)
```

Comment:
Shouldn't we be getting this from `FunctorOps` instead?

Reply:
Well, for that you would need to import all the syntax ops one-by-one...

Reply:
Oh, because they would override each other?!

Reply:
Neither wins; they ambiguate each other.

The fact that `Applicative` is a sub-typeclass of `Functor` isn't known to the compiler, and it sees two applicable implicit conversions from `F[A]` to something with a `map` method, and gives up.

Reply:
That is, if I have both an implicit `Functor` and an implicit `Applicative` in scope, correct?
I guess the issue is a much bigger one then, since we somehow need to tell the compiler that is can extract a functor from an applicative (or rather applicative -> apply -> functor).
I will take the syntax back out. Clearly, more thought needs to be invested here.

Reply:
We tell the compiler that via `BaseHierarchy` right now, whereby it knows that given an `Applicative` it can have a `Functor`.